pipdecode.js

```diff
@@ -0,0 +1,32 @@
+require('buffertools').extend();
```

Comment:
I will try to merge my code with the pipboylib parser and make a PR to the other repo.